influxdb3_write/src/last_cache.rs

```diff
@@ -780,18 +824,60 @@ impl LastCacheStore {
     }
 
     /// Push a [`Row`] from the buffer into this cache
```

Comment:
Need a comment on what the `Option<Vec<(&'a str, DataType)>>` being returned is.